From 7175da4b10d3384748737632d91fbda44138350a Mon Sep 17 00:00:00 2001
From: Jason Gerecke <killertofu@gmail.com>
Date: Tue, 16 Mar 2021 15:26:54 -0700
Subject: [PATCH libwacom 01/10] data: Add multiple Lenovo ThinkPad X1 Yoga Gen
 6 PIDs

Adds the following devices:
  * ISDv4 5276
  * ISDv4 5277
  * ISDv4 5278
  * ISDv4 5279
  * ISDv4 527a
  * ISDv4 527e
  * ISDv4 527f

Signed-off-by: Jason Gerecke <jason.gerecke@wacom.com>
(cherry picked from commit 6ec085291021fe9126ffec507e87c8d6673b9035)
---
 data/isdv4-5276.tablet | 22 ++++++++++++++++++++++
 data/isdv4-5277.tablet | 22 ++++++++++++++++++++++
 data/isdv4-5278.tablet | 22 ++++++++++++++++++++++
 data/isdv4-5279.tablet | 22 ++++++++++++++++++++++
 data/isdv4-527a.tablet | 22 ++++++++++++++++++++++
 data/isdv4-527e.tablet | 22 ++++++++++++++++++++++
 data/isdv4-527f.tablet | 22 ++++++++++++++++++++++
 7 files changed, 154 insertions(+)
 create mode 100644 data/isdv4-5276.tablet
 create mode 100644 data/isdv4-5277.tablet
 create mode 100644 data/isdv4-5278.tablet
 create mode 100644 data/isdv4-5279.tablet
 create mode 100644 data/isdv4-527a.tablet
 create mode 100644 data/isdv4-527e.tablet
 create mode 100644 data/isdv4-527f.tablet

diff --git a/data/isdv4-5276.tablet b/data/isdv4-5276.tablet
new file mode 100644
index 0000000..52549db
--- /dev/null
+++ b/data/isdv4-5276.tablet
@@ -0,0 +1,22 @@
+# Lenovo ThinkPad X1 Yoga Gen 6
+# Sensor Type: AES
+# Features: Touch (Integrated), Tilt
+# HW Resolution: 30159 x 18850 (2540 x 2540 lpi)
+#
+# Autogenerated from sysinfo.gO13VjL2Ut
+# From private bug report
+
+[Device]
+Name=ISDv4 5276
+ModelName=
+DeviceMatch=i2c:056a:5276
+Class=ISDV4
+Width=12
+Height=7
+IntegratedIn=Display;System
+Styli=@isdv4-aes;
+
+[Features]
+Stylus=true
+Touch=true
+Buttons=0
diff --git a/data/isdv4-5277.tablet b/data/isdv4-5277.tablet
new file mode 100644
index 0000000..453e376
--- /dev/null
+++ b/data/isdv4-5277.tablet
@@ -0,0 +1,22 @@
+# Lenovo ThinkPad X1 Yoga Gen 6
+# Sensor Type: AES
+# Features: Touch (Integrated), Tilt
+# HW Resolution: 30165 x 18853 (2540 x 2540 lpi)
+#
+# Autogenerated from sysinfo.rhiB11PAfy
+# From private bug report
+
+[Device]
+Name=ISDv4 5277
+ModelName=
+DeviceMatch=i2c:056a:5277
+Class=ISDV4
+Width=12
+Height=7
+IntegratedIn=Display;System
+Styli=@isdv4-aes;
+
+[Features]
+Stylus=true
+Touch=true
+Buttons=0
diff --git a/data/isdv4-5278.tablet b/data/isdv4-5278.tablet
new file mode 100644
index 0000000..dd86c26
--- /dev/null
+++ b/data/isdv4-5278.tablet
@@ -0,0 +1,22 @@
+# Lenovo ThinkPad X1 Yoga Gen 6
+# Sensor Type: AES
+# Features: Touch (Integrated), Tilt
+# HW Resolution: 30159 x 18850 (2540 x 2540 lpi)
+#
+# Autogenerated from sysinfo.jlmKxHDD9N
+# From private bug report
+
+[Device]
+Name=ISDv4 5278
+ModelName=
+DeviceMatch=i2c:056a:5278
+Class=ISDV4
+Width=12
+Height=7
+IntegratedIn=Display;System
+Styli=@isdv4-aes;
+
+[Features]
+Stylus=true
+Touch=true
+Buttons=0
diff --git a/data/isdv4-5279.tablet b/data/isdv4-5279.tablet
new file mode 100644
index 0000000..12a1d6c
--- /dev/null
+++ b/data/isdv4-5279.tablet
@@ -0,0 +1,22 @@
+# Lenovo ThinkPad X1 Yoga Gen 6
+# Sensor Type: AES
+# Features: Touch (Integrated), Tilt
+# HW Resolution: 30159 x 18850 (2540 x 2540 lpi)
+#
+# Autogenerated from sysinfo.8euIJDBa1t
+# From private bug report
+
+[Device]
+Name=ISDv4 5279
+ModelName=
+DeviceMatch=i2c:056a:5279
+Class=ISDV4
+Width=12
+Height=7
+IntegratedIn=Display;System
+Styli=@isdv4-aes;
+
+[Features]
+Stylus=true
+Touch=true
+Buttons=0
diff --git a/data/isdv4-527a.tablet b/data/isdv4-527a.tablet
new file mode 100644
index 0000000..f46da56
--- /dev/null
+++ b/data/isdv4-527a.tablet
@@ -0,0 +1,22 @@
+# Lenovo ThinkPad X1 Yoga Gen 6
+# Sensor Type: AES
+# Features: Touch (Integrated), Tilt
+# HW Resolution: 30160 x 18850 (2540 x 2540 lpi)
+#
+# Autogenerated from sysinfo.lhwHw4oxV0
+# From private bug report
+
+[Device]
+Name=ISDv4 527a
+ModelName=
+DeviceMatch=i2c:056a:527a
+Class=ISDV4
+Width=12
+Height=7
+IntegratedIn=Display;System
+Styli=@isdv4-aes;
+
+[Features]
+Stylus=true
+Touch=true
+Buttons=0
diff --git a/data/isdv4-527e.tablet b/data/isdv4-527e.tablet
new file mode 100644
index 0000000..2466b9b
--- /dev/null
+++ b/data/isdv4-527e.tablet
@@ -0,0 +1,22 @@
+# Lenovo ThinkPad X1 Yoga Gen 6
+# Sensor Type: AES
+# Features: Touch (Integrated), Tilt
+# HW Resolution: 30159 x 18850 (2540 x 2540 lpi)
+#
+# Autogenerated from sysinfo.azexD4AUXu
+# From private bug report
+
+[Device]
+Name=ISDv4 527e
+ModelName=
+DeviceMatch=i2c:056a:527e
+Class=ISDV4
+Width=12
+Height=7
+IntegratedIn=Display;System
+Styli=@isdv4-aes;
+
+[Features]
+Stylus=true
+Touch=true
+Buttons=0
diff --git a/data/isdv4-527f.tablet b/data/isdv4-527f.tablet
new file mode 100644
index 0000000..25335be
--- /dev/null
+++ b/data/isdv4-527f.tablet
@@ -0,0 +1,22 @@
+# Lenovo ThinkPad X1 Yoga Gen 6
+# Sensor Type: AES
+# Features: Touch (Integrated), Tilt
+# HW Resolution: 30159 x 18850 (2540 x 2540 lpi)
+#
+# Autogenerated from sysinfo.KNSxghuR5y
+# From private bug report
+
+[Device]
+Name=ISDv4 527f
+ModelName=
+DeviceMatch=i2c:056a:527f
+Class=ISDV4
+Width=12
+Height=7
+IntegratedIn=Display;System
+Styli=@isdv4-aes;
+
+[Features]
+Stylus=true
+Touch=true
+Buttons=0
-- 
2.31.1

